python-package/src/opaqueprompts/opaqueprompts_service.py

```diff
@@ -185,7 +186,10 @@ def _send_request_to_opaqueprompts_service(
             response = _session.request(
                 "POST",
                 f"httpa://{hostname}:{port}/{endpoint}",
-                headers={"Authorization": f"Bearer {api_key}"},
+                headers={
+                    "Authorization": f"Bearer {api_key}",
+                    "Request-Version": metadata.version("opaqueprompts"),
```

Comment:
Is it better to rename it to `Client-Version` or `PythonClientVersion` to make clear that this is a version of client instead of server api?

Reply:
Yah - I was uncertain about the name, I was envisioning request version as in the version of the thing making the request but one of those other options might be cleaner. Package-Version could also work. Anyone else have thoughts on the name?

Reply:
I like `Python-Client-Version`

Reply:
My vote is for `Client-Version`.

Reply:
I agree with Octavian.

Reply:
Glad to see a split vote here, the one argument in favor of Python is this is specifically the version of the OpaquePrompts python package - if we had a different client type eventually they wouldn't be on the same version.

Reply:
Going to go with that for now and if people disagree can change it tommorow.

Reply:
So then what, we're going to add different branching in the service depending on if we receive Python-Client-Version, CLI-Client-Version (example), etc...? I think that's too complicated and don't see any case where two clients on the same version wouldn't follow the same logic.

Reply:
Likely at that point is when we would actually properly define versions in other parts of the app (like I brought up on slack), and then we would pivot to using an API version instead so that it is standardized.